src/modules/75-cf/pages/feature-flags/components/__tests__/TagFilter.test.tsx

```diff
@@ -15,81 +14,111 @@ import mockTagsPayload from '../../__tests__/data/mockTagsPayload'
 
 const renderComponent = (props: Partial<TagFilterProps>): RenderResult =>
   render(
-    <Formik initialValues={{}} onSubmit={jest.fn()} formName="tag-filter">
-      <TestWrapper
-        path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
-        pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
-      >
-        <TagFilter
-          disabled={false}
-          tagsData={mockTagsPayload.tags}
-          onTagSearch={jest.fn()}
-          onFilterChange={jest.fn()}
-          tagFilter={[]}
-          {...props}
-        />
-      </TestWrapper>
-    </Formik>
+    <TestWrapper>
+      <TagFilter
+        disabled={false}
+        tagsData={mockTagsPayload.tags}
+        onTagSearch={jest.fn()}
+        onFilterChange={jest.fn()}
+        tagFilter={[]}
+        {...props}
+      />
+    </TestWrapper>
   )
 
 describe('TagFilter', () => {
-  test('it should return the correct tag name when searched', async () => {
-    renderComponent({ tagsData: mockTagsPayload.tags, disabled: false, tagFilter: [{ label: 't1', value: 't1' }] })
-
-    const tagsDropdown = screen.getByRole('textbox')
+  test('it should display a list of tags', async () => {
+    renderComponent({
+      tagsData: mockTagsPayload.tags,
+      disabled: false,
+      tagFilter: [],
+      onFilterChange: jest.fn()
+    })
 
-    await userEvent.type(tagsDropdown, mockTagsPayload.tags[0].name)
+    await userEvent.click(screen.getByText('tagsLabel'))
 
-    expect(tagsDropdown).toHaveValue(mockTagsPayload.tags[0].name)
+    mockTagsPayload.tags.forEach(tag => {
+      expect(screen.getByText(tag.name)).toBeInTheDocument()
+    })
```

Comment:
Nothing wrong here, but it's a good practice in tests to use a `for of` instead of `forEach`. `forEach` doesn't wait for async things and the test will potentially complete before the promises resolve (or reject). A `for of` can wait, so it's worth using to get into the habit.

*100% not an issue in this case*, but worth building up the muscle memory in tests.
```suggestion
    for (const tag of mockTagsPayload.tags ){
      expect(screen.getByText(tag.name)).toBeInTheDocument()
    }
```